config/default.yml

```diff
@@ -3421,6 +3421,11 @@ Style/FrozenStringLiteralComment:
     - never
   SafeAutoCorrect: false
 
+Style/GlobalMethod:
```

Comment:
I'm not sure whether "global" is the right terminology for this. I prefer to think of those as "top-level" method definitions. @rubocop/rubocop-core Ideas how to name this are welcome.

Reply:
> I prefer to think of those as "top-level" method definitions. 

I agree with this. `TopLevelMethodDefinitions` cop may be clear for the term "top-level". There may be other better names.